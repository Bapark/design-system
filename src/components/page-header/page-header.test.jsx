import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import PageHeader from './page-header.jsx';

describe('PageHeader', () => {
  it('has defaults', () => {
    const tree = renderer
      .create((
        <PageHeader title="Defaults" />
      )).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('has a title', () => {
    render(<PageHeader title="Another title" />);
    expect(screen.getByText('Another title').tagName).toEqual('H1');
  });
});
