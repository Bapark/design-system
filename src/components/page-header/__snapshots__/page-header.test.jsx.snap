// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader has defaults 1`] = `
<header>
  <h1>
    Defaults
  </h1>
</header>
`;
